app/scripts/templates/settings/client_disconnect.mustache

```diff
@@ -9,20 +9,28 @@
       </p>
 
       <form novalidate>
-
-        <div class="select-row-wrapper">
-        <label class="label-helper"></label>
-        <div class="select-row">
-          <select class="disconnect-reasons">
-            <option value="reason">{{#t}}Reason for disconnecting…{{/t}}</option>
-            <option value="suspicious">{{#t}}Suspicious device{{/t}}</option>
-            <option value="lost">{{#t}}Lost or stolen device{{/t}}</option>
-            <option value="old">{{#t}}Old or replaced device{{/t}}</option>
-            <option value="duplicate">{{#t}}Duplicate device{{/t}}</option>
-            <option value="no">{{#t}}Rather not say{{/t}}</option>
-          </select>
-        </div>
-        </div>
+          <p class="disconnect-reasons fxa-radio">
+            <label>
+              <input id="suspicious" type="radio" name="disconnect-reasons" value="suspicious">
+              {{#t}}Suspicious device{{/t}}
+            </label><br/>
```

Comment:
We should avoid using `<br/>` for this. Instead we can define a wrapper CSS class, such as `fxa-radio-group label` and make sure `label` inside of that class is always on a new line 👍